### PR TITLE
APP-595 Convert healtcheck endpoint to actuator

### PR DESCRIPTION
**Pull request by robson-sanchez**
- Include the 'spring-boot-actuator' dependency
- Created custom health indicators for applications

### CODE REVIEW

**Review by miltongq — APPROVED** [review]
Nice change making BaseIntegration implement Integration, the code is cleaner after this.
LGTM

File: integration-commons/src/main/java/org/symphonyoss/integration/Integration.java [hunk 1]
```diff
@@ -28,7 +29,7 @@
  *
  * Created by Milton Quilzini on 04/05/16.
  */
-public interface Integration {
+public interface Integration extends HealthIndicator {
```
Comment (dnsymphony) [inline integration-commons/src/main/java/org/symphonyoss/integration/Integration.java]: Hmmm.  This forces everyone who implements Integration to have a dependency on Spring.  We should wrap the Spring classes in our own classes in public APIs (like this interface).

**Review by dnsymphony — CHANGES_REQUESTED** [review]

File: integration-commons/src/main/java/org/symphonyoss/integration/Integration.java (hunk 1 above)
Comment (robson-sanchez) [reply integration-commons/src/main/java/org/symphonyoss/integration/Integration.java]: @dnsymphony Makes sense. I removed the dependency from HealthIndicator interface and created an adapter on integration-healthcheck module to get the health information from Integration interface and adapts to Health interface required by Spring Boot Actuator.

**Comment by robson-sanchez** [comment]
@dnsymphony please check again.

**Review by dnsymphony — APPROVED** [review]

**Pull request merged**